File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/SimpleDependency_ViewModelTestSetupImpl.cs
/// <filename>
///     SimpleDependency_ViewModelTestSetupImpl.cs
/// </filename>
using dependencies;

namespace dependencies
{
    internal class SimpleDependency_ViewModelTestSetupImpl : dependencies.SimpleDependency_ViewModelTestSetup
    {
        public override void Init()
        {
        }

        public override dependencies.SimpleDependency_ViewModel BuildSut()
        {
            return null;
        }

        public override dependencies.MessageBus getMessageBus()
        {
            return null;
        }

        public override dependencies.Logger getLogger()
        {
            return null;
        }
    }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/SimpleDependency_ViewModelTestSetupImpl.cpp
#include "SimpleDependency_ViewModelTestSetupImpl.h"
#include <memory>
#include "SimpleDependency_ViewModel.h"
#include "MessageBus.h"
#include "Logger.h"

namespace dependencies
{
  void SimpleDependency_ViewModelTestSetupImpl::Init()
  {
  }
  std::shared_ptr<SimpleDependency_ViewModel> SimpleDependency_ViewModelTestSetupImpl::BuildSut()
  {
    return nullptr;
  }
  std::shared_ptr<MessageBus> SimpleDependency_ViewModelTestSetupImpl::getMessageBus()
  {
    return nullptr;
  }
  std::shared_ptr<Logger> SimpleDependency_ViewModelTestSetupImpl::getLogger()
  {
    return nullptr;
  }
}
